ql/src/java/org/apache/hadoop/hive/ql/parse/CalcitePlanner.java

```diff
@@ -1830,6 +1850,26 @@ public RelNode apply(RelOptCluster cluster, RelOptSchema relOptSchema, SchemaPlu
       );
       perfLogger.PerfLogEnd(this.getClass().getName(), PerfLogger.OPTIMIZER, "Calcite: Druid transformation rules");
 
+      // 11. Apply Jdbc transformation rules
+      if (LOG.isDebugEnabled()) {
+        LOG.debug("Original plan for jdbc rules:" + System.lineSeparator() + RelOptUtil.toString(calciteOptimizedPlan));
```

Comment:
Could you remove this line? Currently we do not print intermediate plan for any step. I will create a follow-up JIRA and maybe we can add it at the trace level for every step.